src/index.html

```diff
@@ -6,9 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo-container">
+        <a href="#">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <div class="nav-container">
+        <nav class="nav">
+          <ul>
+            <li>
+              <a class="is-active" href="#">Apple</a>
+            </li>
+            <li>
+              <a href="#">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li>
              <a class="is-active" href="#">Apple</a>
            </li>
            
            <li>
              <a href="#">Samsung</a>
            </li>
```
add empty line beetween siblings

Reply:
Я не буду цього робити бо інші ментори потім просять забрати ці пусті строки.

Reply:
привіт, якщо це одностроковий елемент, то не потрбіно додавати пустий рядок, якщо багатостроковий то краще додавити для читабельності